packages/opentelemetry-cloud-monitoring-exporter/__snapshots__/instrument-snapshot.test.ts.js

```diff
@@ -1,28 +1,7 @@
 exports['MetricExporter snapshot tests Counter - DOUBLE 1'] = [
   {
-    "uri": "/v3/projects/otel-starter-project/metricDescriptors",
-    "body": {
-      "type": "workload.googleapis.com/mycounter",
-      "description": "counter description",
-      "displayName": "mycounter",
-      "metricKind": "CUMULATIVE",
-      "valueType": "DOUBLE",
-      "unit": "{myunit}",
-      "labels": [
-        {
-          "key": "string",
-          "description": ""
-        },
-        {
-          "key": "int",
-          "description": ""
-        },
-        {
-          "key": "float",
-          "description": ""
-        }
-      ]
-    },
+    "uri": "/v3/projects/otel-starter-project/metricDescriptors/workload.googleapis.com/mycounter",
```

Comment:
Can you update https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/e0c6c3ee1ec2abe0496b362075f7641fbcad4710/packages/opentelemetry-cloud-monitoring-exporter/test/instrument-snapshot.test.ts#L83-L90 so that it calls GetMetricDescriptor is successfull?